user_guide/configuration/suites.rst

```diff
@@ -103,22 +161,36 @@ criteria. The Gherkin parser comes bundled with a set of cool filters
 such as *tags* and *name* filters. You can use these filters to run
 features with specific tag (or name) in specific contexts:
 
-.. code-block:: yaml
-
-    # behat.yml
-
-    default:
-        suites:
-            web_features:
-                paths:    [ '%paths.base%/features' ]
-                contexts: [ WebContext ]
-                filters:
-                    tags: '@web'
-            api_features:
-                paths:    [ '%paths.base%/features' ]
-                contexts: [ ApiContext ]
-                filters:
-                    tags: '@api'
+.. code-block:: php
+
+    <?php
+    // behat.php
+
+    use App\Tests\Behat\Context\ApiContext;
+    use App\Tests\Behat\Context\WebContext;
+    use Behat\Config\Config;
+    use Behat\Config\Filter\TagFilter;
+    use Behat\Config\Profile;
+    use Behat\Config\Suite;
+
+    $profile = (new Profile('default'))
+        ->withSuite(
+            (new Suite('web_features'))
+                ->withFilter(new TagFilter('@web'))
+                ->withPaths('%paths.base%/features')
+                ->withContexts(WebContext::class)
+        )
+        ->withSuite(
+            (new Suite('api_features'))
+                ->withFilter(new TagFilter('@api'))
+                ->withPaths('%paths.base%/features')
+                ->withContexts(ApiContext::class)
+        )
+    ;
+
+    return (new Config())
+        ->withProfile($profile)
+    ;
 
 .. note::
```

Comment:
This note about the `@` character is no longer required

Reply:
removed